### ISSUE TITLE
Update RESTful servlet to only allow connections from localhost

### ISSUE THREAD

**Issue by dchambers** [BladeRunnerJS/brjs#1260]
At the moment it's possible for one developer to interact with the dashboard and dashboard services on another developer's machine. By updating _dashboard-services_ to only accept connections from `localhost` (using [`getRemoteAddr()`](http://docs.oracle.com/javaee/6/api/javax/servlet/ServletRequest.html#getRemoteAddr())), we can easily prevent this from being misused.

**Comment by janhancic** [comment]
I'm not sure that's a good idea. We often have to access our machines from other computers. In demos for instance. We also often access them during development (let me show you this bug I'm seeing type situations).

What brought about this requirement?

**Comment by andyberry88** [comment]
@janhancic We're only talking about the REST API for the dashboard not the whole of the app server. So any application hosted using the dev server will still be accessible from any machine, but the REST API cannot be used to create apps etc. from anything thats not localhost.

**Comment by janhancic** [comment]
Will I still be able to load the dashboard (& workbenches) though?

**Comment by andyberry88** [comment]
Yes. But any features via the dashboard (create app, create blade, run tests, download war) won't be available.

**Comment by janhancic** [comment]
Cool, sounds good then :+1
